Exchange-Guide.md

```diff
@@ -0,0 +1,196 @@
+# Exchange Guide (WIP)
+
+## Goal
```

Comment:
To be consistent with the other files, and for improved readability of the raw markdown, it'd be nice to have a blank line after all headings like this.